lib/traits/ReattachableView.js

```diff
@@ -23,13 +21,13 @@ var ReattachableView = {
             return;
         }
 
-        var $viewElement = $("[" + IDENTITY_ATTRIBUTE + "='" + this.uid + "']");
+        var viewElement = ViewsFromServer.get(this.uid);
```

Comment:
instead of looking up in the dom for every view, look up from a cached list of reattachable elements